src/fides/api/ops/api/v1/endpoints/privacy_notice_endpoints.py

```diff
@@ -30,6 +30,28 @@
 
 router = APIRouter(tags=["Privacy Notice"], prefix=urls.V1_URL_PREFIX)
 
+DataUseMap = Dict[str, List[schemas.PrivacyNoticeResponse]]
+
+
+def generate_notice_query(
+    db: Session,
+    show_disabled: Optional[bool] = True,
+    systems_applicable: Optional[bool] = False,
+    region: PrivacyNoticeRegion = None,
+) -> Query:
+    """
+    Helper function to generate a `PrivacyNotice` query based on certain parameterizations
+    """
+    notice_query = db.query(PrivacyNotice)
+    if not show_disabled:
+        notice_query = notice_query.filter(PrivacyNotice.disabled.is_(False))
+    if region is not None:
+        notice_query = notice_query.filter(PrivacyNotice.regions.contains([region]))
+    if systems_applicable:
+        data_uses = System.get_system_data_uses(db)
+        notice_query = notice_query.filter(PrivacyNotice.data_uses.overlap(data_uses))  # type: ignore
+    return notice_query
```

Comment:
Nice refactor of this existing bit to reuse here in a slightly different context!